src/ui/components/EmptyView/EmptyView.tsx

```diff
@@ -21,3 +22,36 @@ export function EmptyView({
     </FillView>
   );
 }
+
+export function EmptyView2(
```

Comment:
😄

Reply:
I think current name conveys better that we plan to deprecate the first one